app/web/features/communities/discussions/CommentForm.tsx

```diff
@@ -93,9 +94,11 @@ function _CommentForm(
           name="content"
         />
         <div className={classes.buttonsContainer}>
-          {hideable && <Button onClick={onClose}>{CLOSE}</Button>}
+          {hideable && (
+            <Button onClick={onClose}>{t("communities:close")}</Button>
```

Comment:
```suggestion
            <Button onClick={onClose}>{t("global:close")}</Button>
```
And then perhaps we can delete "close" in the communities namespace altogether for generic strings like these.

Reply:
resolved with 1e062af047a61340e4061de589b87be6c5d5cba3 (sorry for the wrong commit message, I wrote reply instead of close)